aas-gui/Frontend/aas-web-gui/src/components/SubmodelPlugins/CarbonFootprint.vue

```diff
@@ -0,0 +1,180 @@
+                        <g id="SVGRepo_iconCarrier">
+                            <g>
+                                <g>
+                                    <g>
+                                        <path :style="{ fill: 'url(#gradient)' }"
+                                            d="M184.231,239.9c43.9,23.3,64.5,75.4,49.8,123.3c-2.7,8.6-5.1,17.5-7,26.4c-7.8,40,2.7,90.2,46.7,99.1 c22.2,4.7,44.3-3.9,59.9-20.2c46.5-50.5,32.3-202.6,31.9-240.7c0-16.7-0.4-33.8-7-49.4c-14.8-35-56.4-49-93.3-58.7 c-60.6-22.1-98.8,5.8-111.6,26.4S131.531,212.5,184.231,239.9z">
+                                        </path>
+                                        <ellipse cx="141.831" cy="44.3" rx="31.5" ry="44.3"></ellipse>
+                                        <ellipse cx="214.631" cy="51.7" rx="18.7" ry="26.4"></ellipse>
+                                        <ellipse cx="272.931" cy="68" rx="16.7" ry="23.3"></ellipse>
+                                        <ellipse cx="326.131" cy="81.7" rx="14.8" ry="20.6"></ellipse>
+                                        <ellipse transform="matrix(-0.2014 0.9795 -0.9795 -0.2014 546.9331 -227.4809)"
+                                            cx="366.2" cy="109.218" rx="18.3" ry="13.2"></ellipse>
+                                    </g>
+                                </g>
+                            </g>
+                        </g>
+                    </svg>
+                </v-col>
+            </v-row>
+        </v-card>
+    </v-container>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+import { useTheme } from 'vuetify';
+import { useNavigationStore } from '@/store/NavigationStore';
+import { useAASStore } from '@/store/AASDataStore';
+import RequestHandling from '../../mixins/RequestHandling';
+import SubmodelElementHandling from '../../mixins/SubmodelElementHandling';
+
+const PRODUCT_CARBON_FOOTPRINT = 'ProductCarbonFootprint';
+const TRANSPORT_CARBON_FOOTPRINT = 'TransportCarbonFootprint';
+const CO2_EQ = 'CO2eq';
+
+export default defineComponent({
+    name: 'CarbonFootprint',
+    components: {
+        RequestHandling, // Mixin to handle the requests to the AAS
+    },
+    mixins: [RequestHandling, SubmodelElementHandling],
+    props: ['submodelElementData'],
+    setup() {
+        const theme = useTheme();
+        const navigationStore = useNavigationStore();
+        const aasStore = useAASStore();
+
+        return {
+            theme, // Theme Object
+            navigationStore, // NavigationStore Object
+            aasStore, // AASStore Object
+        };
+    },
+    data() {
+        return {
+            productCarbonFootprintData: {} as Record<string, any>,
+            transportCarbonFootprintData: {} as Record<string, any>,
+            productIsoValue: '' as string,
+            transportENValue: '' as string,
+        };
+    },
+    mounted() {
+        // console.log('Component Mounted. SubmodelElementData:', this.submodelElementData);
```

Comment:
calculateSubmodelElementPathes() is not executed anymore. This means that the ConceptDescriptions for the Submodel can't be fetched anymore